gpu-operator/dra-driver.rst

```diff
@@ -0,0 +1,303 @@
+
+###################################
+NVIDIA GPU DRA Driver
+###################################
+
+*******************************
+About the NVIDIA GPU DRA Driver
+*******************************
+
+The NVIDIA GPU DRA Driver leverages the Kubernetes Dynamic Resource Allocation (DRA) API to support NVIDIA IMEX channels available in GH200 and GB200 GPUs.
+The NVIDIA GPU DRA Driver creates and manages IMEX channels through the creation of a ComputeDomain custom resource. Use this custom resource to define your resource templates, and then reference the templates within your workload specs.
+
+An IMEX channel is a construct that allows a set of GPUs to directly read and write each other's memory over a high-bandwidth NVLink. 
+The NVLink connection may either be directly between GPUs on the same node or between GPUs on separate nodes connected by an NVSwitch. 
+Once an IMEX channel has been established for a set of GPUs, they are free to read and write each other's memory via extensions to the CUDA memory call APIs.
+
+The ability to support IMEX channels on GH200 and GB200 systems is essential, as they have been designed specifically to exploit the use of IMEX channels to turn a rack of GPU machines (each with a small number of GPUs) into a giant supercomputer with up to 72 GPUs communicating at full NVLink bandwidth. 
+This allows you to get the most use out of your available GPUs without any additional latency burdens.  
+
+Kubernetes Dynamic Resource Allocation (DRA), available as beta in Kubernetes v1.32, is an API for requesting and sharing resources between pods and containers inside a pod. 
+This feature treats specialized hardware as a definable and reusable object and provides the necessary primitives to support cross-node resources such as IMEX channels. 
+Along with the NVIDIA GPU DRA Driver, you are able to use DRA to define IMEX channel resources that can be managed by Kubernetes.
+
+Refer to the `Kubernetes DRA documentation <https://kubernetes.io/docs/concepts/scheduling-eviction/dynamic-resource-allocation/>`__ for details on this feature. 
+
+*************
+Prerequisites
+*************
+
+- GH200 and GB200 GPUs with Mulit-Node NVLink connections between GPUs.
+
+- 565 or later driver installed on all GPU nodes. 
+  Drivers must be pre-installed on all GPU nodes before installing the NVIDIA GPU Operator as operator managed drivers are not supported at this time.
+
+- IMEX packages installed on GPU nodes with systemd service disabled.
+  The IMEX package verisons must match the installed driver version.
+
+  The following example shows how to install IMEX drivers. 
+
+  .. code-block:: console 
+

```

Comment:
These are assuming, you already installed NVIDIA drivers

DRIVER_VERSION=$(nvidia-smi -i 0 --query-gpu=driver_version --format=csv,noheader,nounits)
IMEX_DRIVER_VERSION="${DRIVER_VERSION%%.*}"
TARGETARCH=$(uname -m | sed -E 's/^x86_64$/amd64/; s/^(aarch64|arm64)$/arm64/')

Reply:
yes, line 32 defines the second prerequisite for installing drivers